pkgs/tools/admin/balena-cli/default.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+, installShellFiles
```

```suggestion
, installShellFiles
```
It's declared but not used?